android/Hybridge/project.properties

```diff
@@ -11,5 +11,6 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-13
+target=android-21
 android.library=true
+android.library.reference.1=../../../mca-android-go/xwalk_core_library
```

Comment:
I'd assume XWalk stuff is at the same level as the initial folder for Hybridge. And document it in the README please :grin: